pimegaApp/Db/module.template

```diff
@@ -0,0 +1,104 @@
+#=================================================================#
+# Template file: module.template
+# Database for PIMEGA module-based records
+
+record(waveform, "$(P)$(R)MB_Temperature_RBV") {
```

Comment:
Why `$(P)$(R)` instead of `$(P)$(R)$(MODULE)`?

This would avoid having to deal with recursive substitution details, and I think it would also be more explicit.

Reply:
Even if this was loaded at runtime (instead of being preprocessed at build time), I think I'd be in favor of `MODULE`. Users of this template aren't going to use `P=$(P),R=$(R)` when instantiating the individual template, they will need to use `P=$(P),R=$(R)M1/2/3/4`. At that point, it's already non-standard, and I think a separate variable makes things clearer.

Furthermore, when using substitutions, this makes it clearer as to what values come from the `dbLoadRecords/Template` call, and what values come from the `.substitution` file itself. The module number is a "channel", not the device identifier.

Reply:
I can understand your reasoning, but prefer my version. Let me know if your opinion changes.